src/main/java/org/opentripplanner/gtfs/mapping/TransitModeMapper.java

```diff
@@ -77,4 +81,28 @@ public static TransitMode mapMode(int routeType) {
                 throw new IllegalArgumentException("unknown gtfs route type " + routeType);
         }
     }
+
+    /**
+     * For routes a mode must be set. If {@link #mapMode} returns it as null, we use BUS as
+     * default.
+     *
+     * @param routeType  Original route type that was mapped into a mode
+     * @param route      The route for which the mode is for
+     * @param issueStore Issue store
+     */
+    public static TransitMode mapModeForRoute(
```

Comment:
I thinks it would make sense to move this to `RouteMapper`

Reply:
I was wondering if there was a better place for this. However, would it be weird to use the gtfs RouteMapper from the netex RouteMapper. To me it feels a bit weird that the netex uses methods from the gtfs package but it already does use this file that is under the gtfs package. In the current state, this TransitModeMapper is closely linked to GTFS and therefore it feels reasonable that it's in the gtfs package.

Reply:
We discussed this today and in the future, netex will no longer use this gtfs transit mode mapper. I refactored the code to RouteMapper as suggested.